AmplifyPlugins/Storage/Tests/StorageHostApp/copy_configuration.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
Why is this script needed? Wasn't the config already being pulled in during build?

Reply:
this script is calls by the Build Phase script. this way is not embedded in the Xcode project file.